docs/modules/sort/quick_start.md

```diff
@@ -20,9 +20,7 @@ Now you can submit job to flink with the jar compiled, refer to [how to submit j
 Example：
 ```
 ./bin/flink run -c org.apache.inlong.sort.singletenant.flink.Entrance inlong-sort/sort-[version].jar \
```

Comment:
org.apache.inlong.sort.singletenant.flink.Entrance
->
org.apache.inlong.sort.flink.Entrance

singletenant have no meaning start from this version now, we should rename the entrance class name.